### PR TITLE
add article only flag

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

File: molo/forms/migrations/0009_add_article_form_field.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    operations = [
+        migrations.AddField(
+            model_name='moloformpage',
+            name='article_form_only',
+            field=models.BooleanField(default=False, help_text='This will display only in article pages', verbose_name='Is Article Form Only'),
+        ),
+        migrations.AlterField(
+            model_name='personalisableformfield',
+            name='field_type',
+            field=models.CharField(choices=[('singleline', 'Single line text'), ('multiline', 'Multi-line text'), ('email', 'Email'), ('number', 'Number'), ('url', 'URL'), ('checkbox', 'Checkbox'), ('checkboxes', 'Checkboxes'), ('dropdown', 'Drop down'), ('radio', 'Radio buttons'), ('date', 'Date'), ('datetime', 'Date/time'), ('hidden', 'Hidden field')], max_length=16, verbose_name='field type'),
```
Comment (KaitCrawford) [inline molo/forms/migrations/0009_add_article_form_field.py]: I don't see this change in the models.py. Is it supposed to be here?

**Review by KaitCrawford — COMMENTED** [review]
One small query about the migration

File: molo/forms/migrations/0009_add_article_form_field.py (hunk 1 above)
Comment (moh-moola) [reply molo/forms/migrations/0009_add_article_form_field.py]: Thanks for this... will remove

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Pull request merged**